azure-pipelines.yml

```diff
@@ -149,6 +155,97 @@ jobs:
     env:
       SPARK_HOME: $(Build.BinariesDirectory)\spark-2.4.5-bin-hadoop2.7
 
+  - task: DotNetCoreCLI@2
+    displayName: 'E2E tests for Spark 2.3.1 with oldest compatible worker (0.9.0)'
+    inputs:
+      command: test
+      projects: '**/Microsoft.Spark.E2ETest/*.csproj'
+      arguments: '--configuration $(buildConfiguration)'
+    env:
+      SPARK_HOME: $(Build.BinariesDirectory)\spark-2.3.1-bin-hadoop2.7
+      DOTNET_WORKER_DIR: $(Build.BinariesDirectory)\Microsoft.Spark.Worker\Microsoft.Spark.Worker-0.9.0
```

Comment:
Can we create a variable for the version?

Reply:
Also, is it possible to use the env variable in `displayName`?

Reply:
Looks like I can't use the version name variable in displayName

Reply:
@imback82 Looks like using variable names is not possible anywhere in the yaml text, but only as [separate nodes](https://stackoverflow.com/questions/41620674/use-placeholders-in-yaml)

Reply:
Which combinations have you tried so far? Did you try `$(name)`? For example: https://github.com/dotnet/arcade/blob/master/azure-pipelines-merge-mirror.yml

Reply:
That worked, thank you!

Reply:
0.8.0 failing with the following error - `Caused by: org.apache.spark.api.python.PythonException: System.MissingMethodException: Method not found: 'Microsoft.Spark.Sql.Row Microsoft.Spark.Sql.Row.op_Implicit(Microsoft.Spark.Sql.GenericRow)'.`